src/xenia/vfs/devices/disc_image_device.cc

```diff
@@ -127,9 +127,11 @@ bool DiscImageDevice::ReadEntry(ParseState* state, const uint8_t* buffer,
   entry->attributes_ = attributes | kFileAttributeReadOnly;
   entry->size_ = length;
   entry->allocation_size_ = xe::round_up(length, bytes_per_sector());
-  entry->create_timestamp_ = 0;
-  entry->access_timestamp_ = 0;
-  entry->write_timestamp_ = 0;
+
+  // Set to January 1, 1601 (UTC) in 100-nanosecond intervals
```

Comment:
Ops, my bad. I thought about another point in the code. Already forgot that I've done there. I've added the offset so later time conversion won't crash, yes,